src/FuelClient.cc

```diff
@@ -986,6 +986,14 @@ bool FuelClient::ParseWorldFileUrl(const common::URI &_fileUrl,
     return false;
   }
 
+  std::vector<std::string> tokens = ignition::common::split(file, "/");
+  std::string fileTemp;
+  for (auto s : tokens)
```

Comment:
```suggestion
  for (const auto &s : tokens)
```

Reply:
I see you resolved the conversation but didn't apply the change. Not sure if you think it's not needed or overlooked it?